test/integration/targets/bigip_policy_rule/tasks/issue-01546.yaml

```diff
@@ -3,6 +3,7 @@
 - name: Issue 01546 - Create published policy
   bigip_policy:
     name: issue-01546
+  when: f.system_info.product_version >= "13.0.0"
```

Comment:
is the variable f registered anywhere? these variables do not persists across tasks, also you should just do this conditional on import_task in main.yaml. so change this

Reply:
I have included the statement/playbook for populating f variable:

```
- name: Collect BIG-IP facts
  bigip_device_info:
    include: system-info
  register: f
```

This comment is addressed now.